src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -69,6 +71,7 @@ public abstract class BatchExportProcessor<T> : BaseExportProcessor<T>
             Guard.ThrowIfOutOfRange(exporterTimeoutMilliseconds, min: 0);
 
             this.circularBuffer = new CircularBuffer<T>(maxQueueSize);
+            this.circularBufferDroppedCount = Sdk.SdkInternalStateMeter.CreateObservableCounter($"{typeof(T)}.CircularBufferDroppedCount", () => this.DroppedCount);
```

Comment:
the name can be "BatchExportProcessor.DroppedCount". CircularBuffer is internal impln.

Reply:
I was actually trying to differentiate between logs and spans. The name is getting very big
`OpenTelemetry.Logs.LogRecord.BatchExportProcessor.DroppedCount`
`System.Diagnostics.Activity.BatchProcessor.DroppedCount`

Reply:
Excluded the namespace part.